Fix - Removes ORM from golem ship

Zergspower opened the pull request:
## About The Pull Request

Replaces the Golem ship ORM with a circuit board and parts


## Why It's Good For The Game

Removes the automatic resource link to the station to avoid both accidental deletion of mined materials from the Golems as well as removing the opportunity to steal or purposefully cripple the stations supply.

## Changelog


:cl:
add: Parts and Machine frame to build an ORM from the golem ship
del: Removed Prebuilt ORM from the golem ship
/:cl:

Zergspower committed:
Fix - Removes ORM from golem ship

Replaces the Golem ship ORM with a circuitboard and parts

SuperNovaa41 commented:
Fixes https://github.com/tgstation/tgstation/issues/60944 (i think)

Zergspower commented:
Yep, basically just gets rid of the starting ORM and replaces it with a machine frame and the parts to build it - easy peasy

Fikou commented:
your changelog doesnt tell players anything useful

ilikesaying commented:
changelog should be
add: Parts and Machine frame to build an ORM from the golem ship
del: Removed Prebuilt ORM from the golem ship

Serxule commented:
yeah that changelog sounds like you're just trying to make miners have a harder life

Serxule commented:
> Fixes #60944 (i think)

this was a problem with anything that could be linked (if it wasnt already), fairly sure its been fixed

Zergspower commented:
> yeah that changelog sounds like you're just trying to make miners have a harder life, also, the golem ORM was never connected or linked to the station, only miners can do that, so i dont think this even changes anything

This is the Golem ship not the miners ORM that's on station, it currently exists linked to the stations Ore Silo

Zergspower commented:
> 



> changelog should be add: Parts and Machine frame to build an ORM from the golem ship del: Removed Prebuilt ORM from the golem ship

Updated

SuperNovaa41 commented:
> > Fixes #60944 (i think)
> 
> this was a problem with anything that could be linked (if it wasnt already), fairly sure its been fixed

No



> yeah that changelog sounds like you're just trying to make miners have a harder life, also, the golem ORM was never connected or linked to the station, only miners can do that, so i dont think this even changes anything

The bug i posted is that it does in fact start linked to the station on icebox if it spawns on a certain zlevel

Serxule commented:
> The bug i posted is that it does in fact start linked to the station on icebox if it spawns on a certain zlevel

could easily fix it by just not letting golems spawn on the upper z-level

WarlockD commented:
Wait a sec, how did this happen?  The ORM was never connected originaly?

Zergspower commented:
> Wait a sec, how did this happen? The ORM was never connected originaly?

it was, mostly an icebox issue though rather than lavaland
